libraries/AP_AHRS/AP_AHRS_Logging.cpp

```diff
@@ -159,7 +159,8 @@ void AP_AHRS_View::Write_Rate(const AP_Motors &motors, const AC_AttitudeControl
         yaw_out         : motors.get_yaw()+motors.get_yaw_ff(),
         control_accel   : (float)accel_target.z,
         accel           : (float)(-(get_accel_ef().z + GRAVITY_MSS) * 100.0f),
-        accel_out       : motors.get_throttle()
+        accel_out       : motors.get_throttle(),
+        throttle_slew   : motors.get_throttle_slew_rate()
```

Comment:
We do have a motors log, `MOTB`. But its only 10hz, I guess too slow for what you need?

May still be better to include that under "fast attitude" than have two extra floats in `RATE` as it is enabled by default.

Reply:
Yeah I need to see the slew based on the effect it has on PD and angle P. One extra float? Not sure what you are saying here.

Reply:
Sorry, your right, one extra float. I was just thinking its a little odd having motors stuff in the rate log, but I was forgetting the roll/pitch/yaw outputs.